MPF.Core/Modules/Redumper/Parameters.cs

```diff
@@ -507,6 +516,9 @@ public override void GenerateSubmissionInfo(SubmissionInfo info, Options options
                     info.Artifacts["cue"] = GetBase64(GetFullFile($"{basePath}.cue")) ?? string.Empty;
                 if (File.Exists($"{basePath}.fulltoc"))
                     info.Artifacts["fulltoc"] = GetBase64(GetFullFile($"{basePath}.fulltoc")) ?? string.Empty;
+                // if (File.Exists($"{basePath}.hash"))
+                //     info.Artifacts["hash"] = GetBase64(GetFullFile($"{basePath}.hash")) ?? string.Empty;
+                // // Also: "{basePath} (Track X).hash" (get from cuesheet)
```

Comment:
If these are just lists of hashes, these are okay to be included as encoded artifacts.

Reply:
1. I have included basePath.hash, but including all the possible hash files for a multi-track disc will remain a TODO